store/sqlstore/team_store.go

```diff
@@ -357,16 +357,14 @@ func (s SqlTeamStore) SearchPrivate(term string) store.StoreChannel {
 	})
 }
 
-func (s SqlTeamStore) GetAll() store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var data []*model.Team
-		if _, err := s.GetReplica().Select(&data, "SELECT * FROM Teams ORDER BY DisplayName"); err != nil {
-			result.Err = model.NewAppError("SqlTeamStore.GetAllTeams", "store.sql_team.get_all.app_error", nil, err.Error(), http.StatusInternalServerError)
-			return
-		}
+func (s SqlTeamStore) GetAll() ([]*model.Team, *model.AppError) {
+	var data []*model.Team
```

Comment:
nit: probably makes more sense to rename de variable to `teams` instead of `data`.